### PR TITLE
migrate to Vonage namespace

### PR DESCRIPTION
**Pull request by bencgreenberg**
This PR is dependent on the migration first of the Ruby SDK to the Vonage namespace and its publication to Rubygems.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Nexmo Community
+Copyright (c) 2020 Vonage Community
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply LICENSE.md:3]: Also remove the (c)

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ ruby /path/to/example.rb
 
 ## Request an Example
 
-Please [raise an issue](https://github.com/nexmo-community/nexmo-ruby-quickstart/issues) to request an example that isn't present within
+Please [raise an issue](https://github.com/Nexmo/nexmo-ruby-code-snippets/issues) to request an example that isn't present within
```
Comment (kellyjandrews) [inline README.md:31]: I think this moved to Vonage, correct?

**Review by kellyjandrews — CHANGES_REQUESTED** [review]
Just a couple of things to look at and address. Otherwise looks ok from my side.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
-# Nexmo Quickstart Examples for Ruby
+# Vonage APIs Quickstart Examples for Ruby
 
 <img src="https://developer.nexmo.com/assets/images/Vonage_Nexmo.svg" height="48px" alt="Nexmo is now known as Vonage" />
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply README.md:3]: I'd say swap it with this one - https://developer.nexmo.com/images/logos/vbc-logo.svg

**Review by kellyjandrews — CHANGES_REQUESTED** [review]
Update the image, and I think it's ready to go.

File: README.md (hunk 3 above)
Comment (bencgreenberg) [inline README.md:3]:
```suggestion
<img src=" https://developer.nexmo.com/images/logos/vbc-logo.svg" height="48px" alt="Vonage APIs" />
```

**Review by kellyjandrews — APPROVED** [review]
lgtm

**Pull request merged**